fix(ipam): fixed content issues and applied new best practices (#6742)
scaleway/docs-content: pages/ipam/reference-content/ipv6.mdx

description: Learn about IPv6 at Scaleway - configuration, routing, security, and best practices for seamless integration and scalability. Transition smoothly with our step-by-step guide.
tags: ipv6 ipv4 support
dates:
  validation: 2026-07-29
  posted: 2024-11-18
---

> IPv6 is essential as the internet grows more connected, secure, and scalable. While most traffic still uses IPv4, IPv6 adoption is growing steadily as tech companies and internet service providers (ISPs) switch to the newer standard.

Reviewer: ```suggestion
IPv6 is essential as the internet becomes more connected, secure, and scalable. While most traffic still uses IPv4, IPv6 adoption is growing steadily as tech companies and internet service providers (ISPs) switch to the newer standard.
```

Reviewer: Might be an overkill to change "grows" but since we have "is growing" in the next sentence, they felt too close to each other. I leave it up to you whether or not you switch out "grows" to "becomes", your choice entirely.

This article explains how IPv6 works at Scaleway and how to configure your resources to take advantage of this protocol.

## What is IPv6?

Internet Protocol (IP) enables machines to locate and communicate with each other on networks like the Internet or private subnets, by assigning each connected machine a unique IP address. An IP address is a set of numbers to identify the machine on the network. 

> The most commonly recognized and most widely-used IP version is **IPv4**, launched in 1983. Each IPv4 address has 32 bits. Written in human-readable form, an IPv4 address is generally shown as four octets separated by periods, e.g. `151.115.59.87`. However, with the growing number of machines connected to the internet, the available 4.3 billion unique IPv4 addresses are nearly exhausted. 

Reviewer: ```suggestion
The most commonly recognized and most widely-used IP version is **IPv4**, launched in 1983. Each IPv4 address has 32 bits. Written in human-readable form, an IPv4 address is generally shown as four octets separated by periods, e.g., `151.115.59.87`. However, with the growing number of machines connected to the internet, the available 4.3 billion unique IPv4 addresses are nearly exhausted. 
```

> The most commonly recognized and most widely-used IP version is **IPv4**, launched in 1983. Each IPv4 address has 32 bits. Written in human-readable form, an IPv4 address is generally shown as four octets separated by periods, e.g. `151.115.59.87`. However, with the growing number of machines connected to the internet, available IPv4 addresses are nearly exhausted—only 4.3 billion unique addresses are possible. 

Reviewer 2: ```suggestion
The most commonly recognized and most widely-used IP version is **IPv4**, launched in 1983. Each IPv4 address has 32 bits. Written in human-readable form, an IPv4 address is generally shown as four octets separated by periods, e.g. `151.115.59.87`. However, with the growing number of machines connected to the internet, the available 4.3 billion unique IPv4 addresses are nearly exhausted. 
```

This is where **IPv6** comes in, the most recent version of the IP protocol. Each IPv6 address has 128 bits, meaning a potential pool of 3.4 x 1038 unique IPv6 addresses - you could assign a trillion unique IPv6 addresses to every atom in a human body and still have addresses left over.

> Written in human-readable form, an IPv6 address can be shown as eight groups of four hexadecimal digits, each group representing 16 bits and separated by a colon, e.g. `2001:0DB8:0000:0003:0000:01FF:0000:002E`. This can also be written as `2001:DB8::3:0:1FF:0:2E`.

Reviewer: ```suggestion
Written in human-readable form, an IPv6 address can be shown as eight groups of four hexadecimal digits, each group representing 16 bits and separated by a colon, e.g., `2001:0DB8:0000:0003:0000:01FF:0000:002E`. This can also be written as `2001:DB8::3:0:1FF:0:2E`.
```

> In addition to providing a bigger address space, IPv6 also includes a built-in network security layer (IPsec), as well as improved features for reliability and efficiency, like auto-configuration, streamlined headers and improved Quality of Service (QoS). All leading to a more robust and secure protocol, that can potentially offer lower latency and faster data transfer.

Reviewer: ```suggestion
In addition to providing a bigger address space, IPv6 also includes a built-in network security layer (IPsec), as well as improved features for reliability and efficiency, like auto-configuration, streamlined headers, and improved Quality of Service (QoS). All of these capabilities lead to a more robust and secure protocol, which can potentially offer lower latency and faster data transfer.
```

Reviewer: The last sentence wasn't a full sentence.
Also, "that" is defining/restrictive, and I think here we're just adding some extra info, hence the switch from "that" to "which".
